drivers/src/pci/registers.rs

```diff
@@ -298,6 +328,47 @@ impl RegisterMasks for BridgeControl::Register {
     }
 }
 
+// Hide D1/D2 and PME support for now.
```

Comment:
Should we file a bug to allow power state transitions? For the proof of concept I don't think we need to handle it.

Reply:
Done.